scss/civicrm/common/_fields.scss

```diff
@@ -0,0 +1,7 @@
+.crm-mailing-recipients-row {
```

Comment:
No need to add empty line here as we didn't find any guide for this and most of current cases doesn't apply an empty line in such case